popoolation2.rb

```diff
@@ -0,0 +1,101 @@
+class Popoolation2 < Formula
+  desc "Allows to compare allele frequencies for SNPs between two or more populations and to identify
+        significant differences. PoPoolation2 requires next generation sequencing data of pooled genomic
+        DNA (Pool-Seq). It may be used for measuring differentiation between populations, for genome wide
+        association studies and for experimental evolution."
+  homepage "https://sourceforge.net/p/popoolation2/wiki/Main/"
+  url "https://downloads.sourceforge.net/project/popoolation2/popoolation2_1201.zip"
+  version "1.2.01"
+  sha256 "7f45353a211ec88d36fd1beed1d128673f3ff911b731211eb5a39f8602bddd42"
+
+  bottle :unneeded
+
+  depends_on "cpanminus" => :build
+  depends_on :perl => ["5.8", :run]
+  depends_on :java => ["1.8", :run, :recommended]
+  depends_on "r" => [:run, :optional]
+  depends_on "samtools" => [:run, :optional]
+  depends_on "bwa" => [:run, :optional]
+
+  def install
+    rm "mpileup2sync.pl"
+    chmod 0755, Dir.glob("*.pl")
+    chmod 0755, Dir.glob("export/*.pl")
+    chmod 0755, Dir.glob("indel_filtering/*.pl")
+    mkdir "foo"
+    mv Dir.glob("*.pl"), "foo/"
+    mv Dir.glob("*.jar"), "foo/"
+    mkdir "foo/export"
+    mv Dir["export/*.pl"], "foo/export/"
+    mkdir "foo/indel_filtering"
+    mv Dir["indel_filtering/*.pl"], "foo/indel_filtering/"
+    mkdir "foo/Modules"
+    mv Dir["Modules/*.pm"], "foo/Modules/"
+    mkdir "foo/Modules/Test"
+    mv Dir["Modules/Test/*.pm"], "foo/Modules/Test/"
+    mkdir "bar"
+    system "cpanm", "-L", "bar", "Text::NSP::Measures::2D::Fisher::twotailed"
+    mv "bar/lib/perl5/Text", "foo/Modules/"
+    File.write("ppool2", <<-EOS.undent)
```

Comment:
We are installing a PoPoolation2 formula and the script is called "ppool2"; I would stick to the full name to make it clear "popoolation2" is a good name...

Reply:
I thought it was a bit long, trying "pop2" now, seeing as Robert himself calls the popoolationTE2 as "popte2" in example scripts.

Reply:
PoPoolationTE2 is a java file and it is from the very beginning popte2; nevertheless, PoPoolation2 was always called like that, and there is no shortcut at all (usually, the scripts are in a folder called "popoolation2". Using popoolation2 seems the right way to go at the moment...

Reply:
Ok I renamed it to "popoolation2"